tests/system/providers/google/cloud/datafusion/example_datafusion_async.py

```diff
@@ -233,40 +234,39 @@
         pipeline_name=PIPELINE_NAME,
         instance_name=INSTANCE_NAME,
         task_id="delete_pipeline",
+        trigger_rule=TriggerRule.ALL_DONE,
     )
     # [END howto_cloud_data_fusion_delete_pipeline]
 
     # [START howto_cloud_data_fusion_delete_instance_operator]
     delete_instance = CloudDataFusionDeleteInstanceOperator(
-        location=LOCATION, instance_name=INSTANCE_NAME, task_id="delete_instance"
+        location=LOCATION,
+        instance_name=INSTANCE_NAME,
+        task_id="delete_instance",
+        trigger_rule=TriggerRule.ALL_DONE,
     )
     # [END howto_cloud_data_fusion_delete_instance_operator]
-    #
-    # Add sleep before creating pipeline
-    sleep_30_1 = BashOperator(task_id="sleep_30_1", bash_command="sleep 30")
-
-    # Add sleep before deleting pipeline
-    sleep_30 = BashOperator(task_id="sleep_30", bash_command="sleep 30")
 
-    # Add sleep before starting pipeline
-    sleep_20 = BashOperator(task_id="sleep_20", bash_command="sleep 40")
+    delete_bucket1 = GCSDeleteBucketOperator(
+        task_id="delete_bucket1", bucket_name=BUCKET_NAME_1, trigger_rule=TriggerRule.ALL_DONE
+    )
+    delete_bucket2 = GCSDeleteBucketOperator(
+        task_id="delete_bucket2", bucket_name=BUCKET_NAME_1, trigger_rule=TriggerRule.ALL_DONE
+    )
 
     (
-        create_bucket1
-        >> create_bucket2
+        [create_bucket1, create_bucket2]
```

Comment:
Can we add comments (in both tests) that would separate preparation, test, and tear down section?
https://cwiki.apache.org/confluence/display/AIRFLOW/AIP-47+New+design+of+Airflow+System+Tests

Something like
```python
(
    # TEST SETUP
    create_dataset
    >> create_table
    # TEST BODY
    >> copy_selected_data
    # TEST TEARDOWN
    >> delete_dataset
)
```